### ISSUE TITLE
Configure HTTPS location at typography.com

### ISSUE THREAD

**Issue by literarymachine** [rightsstatements/rightsstatements.github.io#157]
When requesting https://rightsstatements.github.io/en/, I still get a Mixed Content warning for http://rightsstatements.github.io/fonts/445236/D0B47640DD2C49B64.css. This comes from the redirect configured at typography.com:

```
$ curl -i -H "Referer: https://rightsstatements.github.io/en/" https://cloud.typography.com/7018914/6677352/css/fonts.css
HTTP/1.1 302 Moved Temporarily
Server: Apache
Content-Length: 154
Content-Type: text/html
Location: http://rightsstatements.github.io/fonts/445236/D0B47640DD2C49B64.css
ETag: "348246c6e499ebd12050d9761d4cdc23:1521492159"
Expires: Fri, 14 September 2018 07:41:29 GMT
Last-Modified: Mon, 11 Apr 2016 17:47:03 GMT
Cache-Control: must-revalidate, private
Date: Fri, 14 Sep 2018 07:41:29 GMT
Connection: keep-alive
```

@anarchivist could you look into reconfiguring this somehow?

**Comment by anarchivist** [comment]
Yes, I'm following up, but I'm currently locked out of the Typography.com interface; also, @juliafallon is out of the office for the next two weeks.  I'm looking to see whether forcing the Github Pages setup to redirect to HTTPS will fix this.

**Comment by literarymachine** [comment]
>  I'm looking to see whether forcing the Github Pages setup to redirect to HTTPS will fix this.

@anarchivist could you look into this when you get a chance? I'm not an admin of https://github.com/rightsstatements/rightsstatements.github.io/ so I cannot follow [Securing your GitHub Pages site with HTTPS](https://help.github.com/articles/securing-your-github-pages-site-with-https/).

**Comment by anarchivist** [comment]
@literarymachine Sorry, forgot to update this thread. HTTPS is already enforced for GitHub Pages, but it doesn't seem like that change made a difference. I asked @juliafallon to follow up regarding Typography.com but have yet to hear back.

**Comment by anarchivist** [comment]
Fixed in 32c8b45fd986b8d479d5d643f0ebaf0bca2c697a
